django/contrib/postgres/fields/citext.py

```diff
@@ -1,8 +1,20 @@
-from django.db.models import CharField
+from django.db.models import CharField, EmailField, TextField
 
-__all__ = ['CITextField']
+__all__ = ['CIText', 'CICharField', 'CIEmailField', 'CITextField']
 
 
-class CITextField(CharField):
+class CIText:
```

Comment:
What about forms etc. for that field? I'm open to the idea.

The tests for non-postgres are running locally, I didn't embed the field-imports in a try-except-block.